crates/frontend/src/components/table.rs

```diff
@@ -65,8 +91,10 @@ pub fn table(
                                         .filter(|column| !column.hidden)
                                         .map(|column| {
                                             let cname = &column.name;
-                                            let value: String =
-                                                row.get(cname).unwrap_or(&Value::String(String::new())).html_display();
+                                            let value: String = row
+                                                .get(cname)
+                                                .unwrap_or(&Value::String(String::new()))
+                                                .html_display();
```

Comment:
can we rather do something like this:
```suggestion
                                            let value: String = row.get(cname).map(|v| v.html_display()).unwrap_or_default();
```

Reply:
Why the difference?

Reply:
its better memory wise, right
in the default case we just need a String, and thats what would be get directly

else, currently in default case, you are getting a `Value` type which again is converted to `String` by `html_display`

Reply:
Will look into that, can we take it as it is now?